client/src/components/ExerciseInfoList/ExerciseInfoList.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ExerciseInfo from './ExerciseInfo';
+
+const ExerciseInfoList = ({ exerciseList }) => {
+  const anchorMap = exerciseList.map(exercise => (
+    <li key={`Anchor${exercise._id}`}>
+      <a href={`#${exercise.name.replace(/\s/g, '')}`}>{exercise.name}</a>
+    </li>
+  ));
+
+  const infoMap = exerciseList.map(exercise => (
+    <li key={`Info${exercise._id}`}>
+      <ExerciseInfo exercise={exercise} />
+    </li>
+  ));
```

Comment:
just toss these below, no need to make them variables, unless you think it looks cleaner